client/my-sites/earn/memberships/index.jsx

```diff
@@ -450,17 +499,23 @@ class MembershipsSection extends Component {
 						) }
 					>
 						<NoticeAction href={ `/earn/payments-plans/${ this.props.siteSlug }` } icon="create">
-							{ this.props.translate( 'Add a Payment Plan' ) }
+							{ this.props.translate( 'Add a payment plan' ) }
 						</NoticeAction>
 					</Notice>
 				) }
 				{ this.props.query.stripe_connect_success === 'gutenberg' && (
 					<Notice
 						status="is-success"
 						showDismiss={ false }
-						text={ this.props.translate(
-							'Congrats! Your site is now connected to Stripe. You can now close this window, click "Re-check connection" and add your first payment plan.'
-						) }
+						text={
+							isPayments
+								? this.props.translate(
+										'Congrats! Your site is now connected to Stripe. You can now close this window, click "Re-check connection" and add your first product.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.